test/__snapshots__/make.test.ts.snap

```diff
@@ -254,84 +254,84 @@ and then use the logged account to sign the transaction: \`--accountId <your-acc
 `;
 
 exports[`create 'js' 'none' 'js': --js_none_js--contract--babel.config.json 1`] = `
-Array [
+[
   "--js_none_js--contract--babel.config.json",
   "{
-  \\"plugins\\": [
-    \\"near-sdk-js/lib/build-tools/near-bindgen-exporter\\",
-    [\\"@babel/plugin-proposal-decorators\\", {\\"version\\": \\"legacy\\"}]
+  "plugins": [
+    "near-sdk-js/lib/build-tools/near-bindgen-exporter",
+    ["@babel/plugin-proposal-decorators", {"version": "legacy"}]
   ],
-  \\"presets\\": [\\"@babel/preset-typescript\\"]
+  "presets": ["@babel/preset-typescript"]
 }",
 ]
 `;
 
 exports[`create 'js' 'none' 'js': --js_none_js--contract--build.sh 1`] = `
-Array [
+[
   "--js_none_js--contract--build.sh",
   "#!/bin/sh
 
-echo \\">> Building contract\\"
+echo ">> Building contract"
 
 near-sdk-js build src/contract.ts build/hello_near.wasm
 ",
 ]
 `;
 
 exports[`create 'js' 'none' 'js': --js_none_js--contract--deploy.sh 1`] = `
-Array [
+[
   "--js_none_js--contract--deploy.sh",
   "#!/bin/sh
 
 ./build.sh
 
 if [ $? -ne 0 ]; then
-  echo \\">> Error building contract\\"
+  echo ">> Error building contract"
   exit 1
 fi
 
-echo \\">> Deploying contract\\"
+echo ">> Deploying contract"
 
 # https://docs.near.org/tools/near-cli#near-dev-deploy
 near dev-deploy --wasmFile build/hello_near.wasm",
 ]
 `;
 
 exports[`create 'js' 'none' 'js': --js_none_js--contract--package.json 1`] = `
-Array [
+[
   "--js_none_js--contract--package.json",
   "{
-  \\"name\\": \\"hello_near\\",
-  \\"version\\": \\"1.0.0\\",
-  \\"license\\": \\"(MIT AND Apache-2.0)\\",
-  \\"type\\": \\"module\\",
-  \\"scripts\\": {
-    \\"build\\": \\"./build.sh\\",
-    \\"deploy\\": \\"./deploy.sh\\",
-    \\"test\\": \\"echo use integration-tests\\"
+  "name": "hello_near",
+  "version": "1.0.0",
```

Comment:
I wonder if this actually works (quotes not being escaped feels wrong to me)

Reply:
those files are generated automatically, I do not think we should manually tinker with them

Reply:
I agree, but it looks odd to me, so I just flagged it in case that would trigger some alarm in your head